retriever/engines/postgres.py

```diff
@@ -258,13 +258,19 @@ def get_connection(self):
         import psycopg2 as dbapi
 
         self.get_input()
-        conn = dbapi.connect(
+        try:
+            conn = dbapi.connect(
             host=self.opts["host"],
             port=int(self.opts["port"]),
             user=self.opts["user"],
             password=self.opts["password"],
-            database=self.opts["database"],
-        )
+            database=self.opts["database"],)
+        except dbapi.OperationalError as _: 
+            print("Connection with PostgreSQL Failed! Check file ~/.pgpass or try to restart PostgreSQl Server")
```

Comment:
I think we should not print out our own message. dbapi.OperationalError will provide the error message
I think something like this would be enough/(same can apply for mysql)
     
```python
 except dbapi.err.OperationalError as error:
            print(error)
```